sdk/appcenter-crashes/src/test/java/com/microsoft/appcenter/crashes/CrashesTest.java

```diff
@@ -1120,19 +1125,23 @@ public ManagedErrorLog answer(InvocationOnMock invocation) {
         verify(mockChannel, never()).enqueue(any(ManagedErrorLog.class), eq(crashes.getGroupName()), anyInt());
     }
 
+    @SuppressWarnings("ResultOfMethodCallIgnored")
     private ManagedErrorLog testNativeCrashLog(long appStartTime, long crashTime, boolean correlateSession, boolean hasDeviceInfo) throws Exception {
 
-        /* Setup mock for a crash in disk. */
-        File minidumpFile = mock(File.class);
-        when(minidumpFile.getName()).thenReturn("mockFile");
-        when(minidumpFile.isDirectory()).thenReturn(false);
-        when(minidumpFile.lastModified()).thenReturn(crashTime);
+        /* Create minidump sub-folder. */
+        File minidumpSubfolder = new File("./mockFolder");
```

Comment:
Let's use TemporaryFolder rule for the creation of test files and folders in this PR. It has some benefits, like removing all its content after the test automatically.
See: https://github.com/microsoft/appcenter-sdk-android/blob/develop/sdk/appcenter-crashes/src/test/java/com/microsoft/appcenter/crashes/utils/ErrorLogHelperTest.java#L76

Reply:
Fixed